pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -367,12 +307,11 @@ stdenv.mkDerivation (rec {
     inherit browserName;
   } // lib.optionalAttrs gtk3Support { inherit gtk3; };
 
-} //
-# the build system verifies checksums of the bundled rust sources
-# ./third_party/rust is be patched by our libtool fixup code in stdenv
-# unfortunately we can't just set this to `false` when we do not want it.
-# See https://github.com/NixOS/nixpkgs/issues/77289 for more details
-lib.optionalAttrs (lib.versionAtLeast ffversion "72") {
```

Comment:
IIRC we still support 68-esr so this must stay, right?

Reply:
Right. fixed.